src/index.js

```diff
@@ -1,32 +1,11 @@
 import './style.css';
-import ScoreList from './modules/scoremethods.js';
-
-const scorelist = new ScoreList();
-const score = document.querySelector('.table');
-function loadList() {
-  score.innerHTML = '';
-  if (scorelist.scorelist.length !== 0) {
-    scorelist.scorelist.forEach((score) => {
-      const row = document.createElement('tr');
-      const item = document.createElement('th');
-      item.value = `${score.name}: ${score.score}`;
-      row.appendChild(item);
-      score.appendChild(row);
-    });
-  }
-}
-
-function loadSinglescore() {
-  const index = scorelist.scorelist.length - 1;
-  const row = document.createElement('tr');
-  const item = document.createElement('th');
-  item.innerHTML = `${scorelist.scorelist[index].name}: ${scorelist.scorelist[index].score}`;
-  row.appendChild(item);
-  score.appendChild(row);
-}
+import Score from './modules/score.js';
+import loadSinglescore from './modules/renderPage.js';
+import addScore from './modules/addScore.js';
+import getScores from './modules/getScore.js';
 
 function onPageLoad() {
-  loadList();
+
```

Comment:
Just as suggested in the previous comment, kindly replace your regular function with arrow function👍.